sbnanaobj/StandardRecord/classes_def.xml

```diff
@@ -13,7 +13,10 @@
    <version ClassVersion="10" checksum="2908097963"/>
   </class>
 
-  <class name="caf::SRShower" ClassVersion="10">
+  <class name="caf::SRShower" ClassVersion="13">
```

Comment:
Should be able to suppress all the intermediate versions here. Just renumber 13 to 11 and delete the existing 11 and 12.